packages/tocco-ui/src/EditableValue/typeEditors/NumberEdit.js

```diff
@@ -78,6 +78,10 @@ const NumberEdit = props => {
   )
 }
 
+NumberEdit.contextTypes = {
```

Comment:
To change this component properties has nothing to do with storybook in the first place. I expect a separate commit for this.

**This change might affect the entity-browser. Please test everything manually and verify that it still works. Of course you also have to remove intl from the option property. It's always very important to see changes like this in a bigger picture.**

Reply:
I tested all three components and they work as intended.